### ISSUE TITLE
Audio-Join Status wrong

### ISSUE THREAD

**Issue by runout-at** [bigbluebutton/bigbluebutton#9178]
I try to get some statistics out of BBB. Special interest is if users are joined with microphone, just listening or deaf-mute.

in the XML returned from the API i see sometimes users are joined with hasJoinedVoice=tru and isListeningOnly=true. This should not be possible at all. the participant count is correct. It seems the listenerCount count is wrong in such cases.

Even it says recording: true. Which was not true for this meeting

Until now i don't know how reproduce this and I don't know if this might be an API-issue.

Maybe this happens on joining/unjoining audio multiple time

```
...
<createTime>1587322282091</createTime>
<createDate>Sun Apr 19 20:51:22 CEST 2020</createDate>
...
<hasUserJoined>true</hasUserJoined>
<recording>true</recording>
<hasBeenForciblyEnded>false</hasBeenForciblyEnded>
<startTime>1587322282114</startTime>
<endTime>0</endTime>
<participantCount>6</participantCount>
<listenerCount>1</listenerCount>
<voiceParticipantCount>6</voiceParticipantCount>
<videoCount>0</videoCount>
<maxUsers>0</maxUsers>
<moderatorCount>1</moderatorCount>
<attendees>
...
<attendee>
<userID>w_yilcv611gjml</userID>
<fullName>UserName</fullName>
<role>VIEWER</role>
<isPresenter>false</isPresenter>
<isListeningOnly>true</isListeningOnly>
<hasJoinedVoice>true</hasJoinedVoice>
<hasVideo>false</hasVideo>
<clientType>HTML5</clientType>
</attendee>
...

```

**Comment by SamuelWei** [comment]
@runout-at I have the same issue. 
If the user logs in without a microphone, the isListeningOnly attribute always remains set to true, even if the user leaves the audio conference and logs in again with a microphone.

If the user leaves the audio conference (but remains in the meeting), the user continues to be counted as a listener.

**Comment by prlanzarin** [comment]
listenerCount accounting has been fixed as of 2.3-beta.2.
